src/layouts/DashboardLayout/NavBar/NavItem.js

```diff
@@ -45,10 +41,31 @@ const NavItem = ({
   href,
   icon: Icon,
   title,
+  externalLink,
   ...rest
 }) => {
   const classes = useStyles();
 
+  if (externalLink) {
```

Comment:
Can we avoid a separate if-statement here? Could add conditional rendering in the return statement below

Reply:
That would require conditional rendering in two places, anchor tag and button. This increases lines of code though so let me know if we wanna go with 2 conditional renders.

Reply:
Overlooking this as discussed in offline chat with @sauravhiremath